### DISCUSSION TITLE
Proper update path YOURLS (1.7->1.9.2) and PhP (7.4->8.1)?

### DISCUSSION THREAD

**Discussion by 5022205** [YOURLS/YOURLS#3557] [Q&A]
Can someone advise the (a) proper upgrade path for an installation of YOURLS from v1.7 to v1.9.2 to be done in tandem with a PhP update from v7.4 to v8.1? The latter of these updates is the focus/impetus for the both updates. 

It is my understanding that support for PhP 8 was added in an intermediate release between 1.7 and 1.9.2, so it is not surprising that updating only PhP breaks the YOURLS 1.7 installation. 

Updating PhP in addition to copying v1.9.2 files over top of v1.7 files results in the site appearing to load/function but login to admin portal fails. Rolling back both restores all functionality.

Should I do incremental updates of YOURLS and then update PhP somewhere in that process after PhP 8 support was added?

**Comment by LeoColomb** [comment]
The way you described is supposed to be a good one (upgrade PHP then upgrade YOURLS).
What is the error raised?
Has the YOURLS' own upgrade page been shown?
